src/org/opensolaris/opengrok/configuration/messages/RefreshMessage.java

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
I don't know why it disappeared anyway write a documentation note about this new type to:
1. Messages.java (usage)
2. README.txt (message types)

I can update the wiki by myself